holoviews/core/spaces.py

```diff
@@ -509,6 +516,47 @@ def get_nested_streams(dmap):
     return list(set(layer_streams))
 
 
+
+def get_stream_sources(obj, branch=0):
```

Comment:
Sounds reasonable.

What I don't like is that nothing about this mentions streams! It is really ``get_potential_stream_sources`` but I'm thinking the name should really reflect that is is about splitting objects in overlays into a dictionary.